LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -1852,6 +1853,26 @@ private static void GetIteConfigurationsB(ISuperIO superIO, Manufacturer manufac
                         c.Add(new Control("CPU Optional Fan", 4));
                         break;
 
+                    case Model.X870_AORUS_ELITE_WIFI7: // ITE IT8696E
+                    case Model.X870E_AORUS_ELITE_WIFI7: // ITE IT8696E
+                        t.Add(new Temperature("System #1", 0));
+                        t.Add(new Temperature("PCH", 1));
+                        t.Add(new Temperature("CPU", 2));
+                        t.Add(new Temperature("PCIe x16", 3));
+                        t.Add(new Temperature("VRM MOS", 4));
+                        t.Add(new Temperature("EC_TEMP1", 5));
+                        f.Add(new Fan("CPU Fan", 0));
+                        f.Add(new Fan("System Fan #1", 1));
+                        f.Add(new Fan("System Fan #2", 2));
+                        f.Add(new Fan("System Fan #3 / Pump", 3));
+                        f.Add(new Fan("CPU Optional Fan", 4));
```

Comment:
Based off the two models, System Fan 3 does not reference as a pump header. Only 5/6. Are you simply copying this data entry from another board entry already provided?

Reply:
Yes, you're right i fixed the name of Fan #3